documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -141,6 +141,7 @@ jobs:
       - name: build
         env:
           BASE_PATH: '/your-repo-name'
+          NODE_OPTIONS: '--max-old-space-size=4096'
```

Comment:
if we do add this then we'd need to add some description about why
```suggestion
          # you can increase the memory allocated to the heap if you run out of memory while building
          # https://github.com/vitejs/vite/issues/2433
          # NODE_OPTIONS: '--max-old-space-size=4096'
```

Reply:
Error prevented described here : https://github.com/actions/runner-images/issues/70

Reply:
Ok